samza-api/src/main/java/org/apache/samza/operators/MessageStream.java

```diff
@@ -37,7 +36,7 @@
  * <p>
  * A {@link MessageStream} can be transformed into another {@link MessageStream} by applying the transforms in this API.
  *
- * @param <M>  type of {@link MessageEnvelope}s in this stream
+ * @param <M> type of {@link MessageEnvelope}s in this stream
```

Comment:
Sure, let's update the other operator (api and impl) classes as well then. I kept it at double spaces since that's what Yi's code had.